assets/src/scripts/components/graph/index.js

```diff
@@ -0,0 +1,301 @@
+import { axisBottom, axisLeft } from 'd3-axis';
+import { mouse, select } from 'd3-selection';
+import { line as d3Line } from 'd3-shape';
+import { timeFormat } from 'd3-time-format';
+import { transition } from 'd3-transition';
+import { createStructuredSelector } from 'reselect';
+
+import { link } from 'ngwmn/lib/d3-redux';
+import { initCropper } from 'ngwmn/lib/utils';
+import { retrieveWaterLevels } from 'ngwmn/services/state/index';
+
+import {
```

Comment:
The thought was to hide the implementation details of the state in a single module... so for instance in `store.js`, it can just import a single graph reducer and not have to grab them all.

I guess the actions and selectors don't need to be exported in the same way, though, and we could grab them direct from the sub-module. I'm sort of `+0` on that and would be okay with switching it.

Reply:
That makes sense. I'm okay with leaving it.